adam_core/coordinates/cartesian.py

```diff
@@ -14,16 +13,13 @@
     from .keplerian import KeplerianCoordinates
     from .spherical import SphericalCoordinates
 
-__all__ = ["CartesianCoordinates", "CARTESIAN_COLS", "CARTESIAN_UNITS"]
+__all__ = ["CartesianCoordinates"]
 
 CARTESIAN_COLS = {}
```

Comment:
I suspect that these `{CoordinateType}_COLS` dictionaries are also no longer being used, so they might be safe to remove.